src/index.html

```diff
@@ -17,6 +21,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="MOYO Logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav__container">
+        <ul>
+          <li>
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```